src/device/device-management.ts

```diff
@@ -7,10 +7,12 @@ import { VideoContentHint } from '../media/local-video-stream';
 
 export enum ErrorTypes {
   DEVICE_PERMISSION_DENIED = 'DEVICE_PERMISSION_DENIED',
-  CREATE_CAMERA_STREAM_FAILED = 'CREATE_CAMERA_STREAM_FAILED',
-  CREATE_MICROPHONE_STREAM_FAILED = 'CREATE_MICROPHONE_STREAM_FAILED',
+  CREATE_STREAM_FAILED = 'CREATE_CAMERA_STREAM',
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type Constructor<T> = new (...args: any[]) => T;
+
 /**
  * Represents a WCME error, which contains error type and error message.
```

Comment:
We can do this in a later PR, but I think it's kind of weird that we're referencing WCME here, and naming the error class `WcmeError`, especially since WCME doesn't actually call any of the device management APIs.